sdk/redisenterprise/Azure.ResourceManager.RedisEnterprise/api/Azure.ResourceManager.RedisEnterprise.netstandard2.0.cs

```diff
@@ -159,11 +165,52 @@ public partial class RedisEnterprisePrivateEndpointConnectionResource : Azure.Re
 }
 namespace Azure.ResourceManager.RedisEnterprise.Models
 {
+    public partial class Capability
+    {
+        internal Capability() { }
+        public string Name { get { throw null; } }
+        public bool? Value { get { throw null; } }
+    }
+    public partial class ClusterPropertiesEncryptionCustomerManagedKeyEncryption
```

Comment:
This is really a bad name from an anonymous model (maybe) in the swagger.
Could we rename this to `RedisEnterpriseCustomerManagedKeyEncryption`?

Reply:
Renamed